cli/pull.go

```diff
@@ -155,7 +156,13 @@ func parseNameTag(input string) (string, string) {
 }
 
 // pullExample shows examples in pull command, and is used in auto-generated cli docs.
-// TODO: add example
 func pullExample() string {
-	return ""
+	return `$ pouch images ls
+IMAGE ID            IMAGE NAME                           SIZE
+bbc3a0323522        docker.io/library/busybox:latest     2699
+$ pouch pull docker.io/library/redis:alpine
+$ pouch images ls
```

Comment:
I think currently in master, we only support command `pouch images` rather than `pouch images ls`. 
Yes, this part is also shown in INSTALLATION.md, and @Starnop will come with a fix.
Could you update with `pouch images`?